src/components/projects/BelledejourContent.tsx

```diff
@@ -0,0 +1,135 @@
+import { LayoutProps, layout } from "styled-system";
+
+import Grid from "../Grid";
+import React from "react";
+import image1 from "./../../assets/project-page/belledejour/belledejour-1.jpg";
```

Comment:
these import statements theres a minor change:
`"./../../" === "../../"`

Reply:
hmmmmmm

Reply:
I've done this throughout the codebase - lemme put in a ticket to get rid